docs/fastdds/getting_started/simple_python_app/includes/publisher.rst

```diff
@@ -0,0 +1,96 @@
+Write the Fast DDS publisher
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From the workspace, run the following command to download the HelloWorldPublisher.py file.
+
+.. code-block:: bash
+
+    wget -O HelloWorldPublisher.py \
+        https://raw.githubusercontent.com/eProsima/Fast-RTPS-docs/master/code/Examples/Python/HelloWorld/HelloWorldPublisher.py
+
+Now you have the publisher's source code.
+The publisher is going to send 10 publications under the topic `HelloWorldTopic`.
+
+.. literalinclude:: /../code/Examples/Python/HelloWorld/HelloWorldPublisher.py
+  :language: python
+  :linenos:
+
+Examining the code
+""""""""""""""""""
+
+At the beginning of the file we import the Fast DDS Python bindings.
+
+.. literalinclude:: /../code/Examples/Python/HelloWorld/HelloWorldPublisher.py
+   :language: python
+   :lines: 20
+
+and also the Python module generated by Fast-DDS-Gen as described in :ref:`writing_pubsub_python_datatype` section.
+
+
+.. literalinclude:: /../code/Examples/Python/HelloWorld/HelloWorldPublisher.py
+   :language: python
+   :lines: 21
+
+Then, the :class:`WriterListener` class is defined by inheriting from the |DataWriterListener-python-api| class.
+This class overrides the default DataWriter listener callbacks, which allow us to execute routines in case of an event.
+The overridden callback |DataWriterListener::on_publication_matched-python-api|
+allows you to define a series of actions when a new DataReader
+is detected listening to the topic under which the DataWriter is publishing.
+The :func:`info.current_count_change` detects these changes of DataReaders that are matched to the
+DataWriter.
+This is a member in the |MatchedStatus-python-api| structure that allows you to track changes in the status of subscriptions.
```

Comment:
```suggestion
This class overrides the default DataWriter listener callbacks, which allows the execution of routines in case of an event.
The overridden callback |DataWriterListener::on_publication_matched-python-api|
allows the definition of a series of actions when a new DataReader
is detected listening to the topic under which the DataWriter is publishing.
The :func:`info.current_count_change` detects these changes of DataReaders that are matched to the
DataWriter.
This is a member in the |MatchedStatus-python-api| structure that allows tracking changes in the status of subscriptions.
```